### ISSUE TITLE
api/data/source/:id reports 404 when changeset exists

### ISSUE THREAD

**Issue by regan-sarwas** [nationalparkservice/places-api#25]
A 404 error is returned when the changeset does not exist (appropriately), and when the changeset exists, but is not ready since it has not been closed (__503 Service Unavailable__ seems like a better fit).  A different return code is nice since a client (arc2places) may want to respond differently in these two cases.

**Comment by jimmyrocks** [comment]
That does make sense. Is it returning 404 now?

**Comment by regan-sarwas** [comment]
Yes, unless the changeset is closed, and then I get my data as expected.  I would assign this to me if I could.
